### ISSUE TITLE
Can't find the solution for section 2 exercise.

### ISSUE THREAD

**Issue by Uneebzzz** [mosh-hamedani/vidly-mvc-5#166]
As I am learning this language in 2021, and it was taught in 2016. I am having trouble in finding that repository of code. 
Kindly help if anyone know it would be really appreciated.
Thank You.

**Comment by yugalgrg** [comment]
Yeah recently bought the asp.net course from the website but I'm struggling to find the commit section here for the solutions.

**Comment by vin-1** [comment]
If you click on the 'vidly-mvc-5' repository, then the number of commits is shown in the top right hand corner of the table. Currently showing '50 commits'.

**Comment by NMNaufaldo** [comment]
this is the solution you're looking for : https://github.com/mosh-hamedani/vidly-mvc-5/commit/b727a26e1b4b88abe84a8b48208fec537db2ed43

You can find all the commit on this link : https://github.com/mosh-hamedani/vidly-mvc-5/commits/master

**Comment by seanishok01** [comment]
1. Navigate to https://github.com/mosh-hamedani/vidly-mvc-5
2. Click on 50 commits
![image](https://user-images.githubusercontent.com/16938601/119434715-c0518500-bce6-11eb-8114-a8a2c50e7e37.png)
3. scroll to the bottom of the page and select older
![image](https://user-images.githubusercontent.com/16938601/119434809-f0992380-bce6-11eb-9da9-5aeb3b7cd376.png)
4. Scroll down the page until you find section 2 exercise
![image](https://user-images.githubusercontent.com/16938601/119434914-250cdf80-bce7-11eb-807d-f3baa86017ea.png)
5. Click on the solution - section 2 exercise
6. You can view a list of files changed in the commit by selecting 8 changed files and when you click on each file it will navigate to the respective file section. Click on the ... button to explore the file options (View file)
![image](https://user-images.githubusercontent.com/16938601/119435348-efb4c180-bce7-11eb-9519-6e6766a6c300.png)
